### PR TITLE
masterpage: followup (part 2)

### PR DESCRIPTION
**Pull request by eszkadev**
this one requires core change: https://gerrit.libreoffice.org/c/core/+/137869

### CODE REVIEW

File: browser/src/control/Control.Sidebar.js [hunk 1]
```diff
@@ -146,7 +146,8 @@ L.Control.Sidebar = L.Control.extend({
 	},
 
 	changeDeck: function(unoCommand) {
-		app.socket.sendMessage('uno ' + unoCommand);
+		if (unoCommand !== null)
+			app.socket.sendMessage('uno ' + unoCommand);
 		this.setupTargetDeck(unoCommand);
```
Comment (Ashod) [inline browser/src/control/Control.Sidebar.js:151]: If `unoCommand == null`, should we still call `this.setupTargetDeck` ?

**Review by Ashod — APPROVED** [review]
Thanks @eszkadev. See questions please.

File: browser/src/control/Control.Sidebar.js (hunk 1 above)
Comment (eszkadev) [reply browser/src/control/Control.Sidebar.js:151]: yes, we want to reset it

File: browser/src/layer/tile/CanvasTileLayer.js [hunk 2]
```diff
@@ -1623,6 +1623,12 @@ L.CanvasTileLayer = L.Layer.extend({
 		}
 		else if (textMsg.startsWith('status:') || textMsg.startsWith('statusupdate:')) {
 			this._onStatusMsg(textMsg);
+
+			// update tiles and selection because mode could be changed
```
[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/layer/tile/CanvasTileLayer.js:1627]: yes, we send status because it contains slide list and mode so we can refresh slide previews

**Pull request merged**